GVRf/Framework/framework/src/main/java/org/gearvrf/animation/keyframe/BVHImporter.java

```diff
@@ -0,0 +1,319 @@
+package org.gearvrf.animation.keyframe;
+import org.gearvrf.GVRAndroidResource;
+import org.gearvrf.GVRContext;
+import org.gearvrf.animation.GVRAnimation;
+import org.gearvrf.animation.GVRPose;
+import org.gearvrf.animation.GVRSkeleton;
+import org.gearvrf.animation.keyframe.GVRAnimationBehavior;
+import org.gearvrf.animation.keyframe.GVRAnimationChannel;
+import org.gearvrf.animation.keyframe.GVRSkeletonAnimation;
+import org.gearvrf.utility.Log;
+import org.joml.Quaternionf;
+import org.joml.Vector3f;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+
+public class BVHImporter
+{
+    private String mFileName;
+    private final GVRContext mContext;
+    private final ArrayList<String> mBoneNames = new ArrayList();
+    private final ArrayList<Vector3f> mBonePositions = new ArrayList();
+    private final ArrayList<Integer> mBoneParents = new ArrayList();
+
+    public BVHImporter(GVRContext ctx)
```

Comment:
Unused? Javadoc?

Reply:
Work in progress, not tested yet, needs avatar and matching bvh